webpack.config.js

```diff
@@ -118,6 +129,20 @@ if (TARGET !== 'readme') {
         }
     };
 
+    const json = {
+        test: /\.(json5?|ya?ml)$/, // target json, json5, yaml and yml files
```

Comment:
it looks rather weird that in the "json" block, YAML will be supported. Maybe you can add a bit more documentation on why this is a good idea anyhow?

Reply:
Vue language files can be specified in JSON or YAML.

Reply:
is it then better to call them `vueTranslations` instead?

Reply:
FIxed